scripts/exregional_make_ics.sh

```diff
@@ -532,7 +533,7 @@ settings="
  'mosaic_file_target_grid': ${FIXlam}/${CRES}${DOT_OR_USCORE}mosaic.halo$((10#${NH4})).nc,
  'orog_dir_target_grid': ${FIXlam},
  'orog_files_target_grid': ${CRES}${DOT_OR_USCORE}oro_data.tile${TILE_RGNL}.halo$((10#${NH4})).nc,
- 'vcoord_file_target_grid': ${FIXam}/global_hyblev.l65.txt,
+ 'vcoord_file_target_grid': ${FIXam}/global_hyblev_fcst_rrfsL65.txt,
```

Comment:
To match the increase to 65 levels used in RRFS_dev, this specific file needs to be used.